readthedocs_build/config/config.py

```diff
@@ -338,15 +338,21 @@ def set_output_base(self, directory):
             build['output_base'] = os.path.abspath(directory)
 
 
-def load(path, env_config):
+def load(path, env_config, only=False):
     """
     Load a project configuration and all the contained build configs for a
     given path. That is usually the root of the project.
 
     The config will be validated.
+
+    `only` means not to search for config files, but use the one passed in
     """
 
-    config_files = list(find_all(path, CONFIG_FILENAMES))
+    if only:
+        config_files = [path]
```

Comment:
I think the `path` contain the repo path only. not the absolute path of `readthedocs.yml`

Reply:
Yea, this will be fixed on the RTD side.